java/README.md

```diff
@@ -0,0 +1,108 @@
+# jGLOME
+**This is not an officially supported Google product.**
+
+This module contains a Java implementation for the GLOME
+protocol.
+
+## Usage
+
+Let's say Alice and Bob want to establish a communication. Then both parties
+should know their peer's public key. This is an example of how Alice can use 
+the Glome object to generate a tag:
+
+```Java
+import com.google.jglome.Glome;
+import com.google.jglome.Glome.GlomeBuilder;
+
+public class Main {
+
+  public void sendMsgToBob() {
+    // Generates a Glome object for Alice with Bob's public key `BobPublicKey` 
+    // and minimum length of Bob's tag - 32.
+    Glome AliceManager = new GlomeBuilder(BobPublicKey, 32).build();
+
+    // Generates a tag with `msg` message. Considering that this message 
+    // is first to be sent to Bob from Alice, the counter is set to 0.
+    byte[] tag = AliceManager.generateTag(msg, 0);
+    
+    // Post-processing code block.
+  }
+
+}
+```
+
+Then Alice should send Bob both `msg`, `tag` and Alice's public key.
+As soon as Bob receives them he will need to do the following:
```

Comment:
I try to avoid pronouns such as he/she but that's personal preference.

Reply:
What is better to be used instead? Or nothing and the paragraph is better be paraphrased?

Reply:
One way to construct this differently would be "When receiving them, Bob will need to do the following...". Another option is to use 'they' instead.

Reply:
Maybe "As soon as the information is received by Bob, the following can be be done to verify it:"